week1/Q2.py

```diff
@@ -0,0 +1,58 @@
+# element of a linked list
+# consists a value of itself and a link to the next element
+class Node:
+	def __init__(self, value = None, next = None):
+		self.value = value
+		self.next = next
+
+# data structure, consists of nodes
+# clear -- resets a linked list
+# add -- adds new node, adds a link to the new element to the last one (if present)
+class LinkedList:
```

Comment:
If the comments are formatted as docstrings (see https://www.python.org/dev/peps/pep-0257/) there are tools that will make use of them (like `help(LinkedList)` will provide access to your docstring).